Dockerfile

```diff
@@ -6,5 +6,8 @@ EXPOSE 6000
 WORKDIR /services
 ADD . /services
 
+RUN useradd -ms /bin/bash services_user
+USER services_user
```

Comment:
According [to the docs](https://docs.docker.com/engine/reference/builder/#user) you also need to provide a group otherwise it will default to `root`.